packages/ui/components/form/inputs/Input.tsx

```diff
@@ -348,7 +348,8 @@ const PlainForm = <T extends FieldValues>(props: FormProps<T>, ref: Ref<HTMLForm
           event.stopPropagation();
 
           if (form.formState?.errors?.apiError) {
-            form.clearErrors("root");
+            // @ts-expect-error FIXME Fix these typings
+            form.clearErrors("apiError");
```

Comment:
<img width="1050" alt="Screenshot 2023-04-26 at 6 31 19 PM" src="https://user-images.githubusercontent.com/1780212/234582849-05114545-5354-46e5-ade1-f2463d772bb7.png">
This is a better place to handle `apiError` as it is added by Change Password logic and doesn't make sense in generic Input component.

I think we should remove the complete condition at line 350.